packages/config/src/providers/envs/EnvsConfigSource.ts

```
@@ -0,0 +1,23 @@
+export interface EnvsConfigSourceOptions {
+  parseJson?: boolean;
+}
+
+export class EnvsConfigSource<Config extends EnvsConfigSourceOptions = EnvsConfigSourceOptions> implements ConfigSource<Config> {
+  options: Config;
+
+  getAll(): Record<string, unknown> {
+    const {parseJson = true} = this.options;
+
+    return Object.entries(process.env).reduce((obj, [key, value]) => {
+      const formattedValue = parseJson ? jsonParse(value!) : value;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)